src/Core/Plugin.php

```diff
@@ -318,14 +318,8 @@ public static function bootstrap($name)
         if (!$plugin->isEnabled('bootstrap')) {
             return false;
         }
-        // Disable bootstrapping for this plugin as it will have
-        // been bootstrapped.
-        $plugin->disable('bootstrap');
 
-        return static::_includeFile(
-            $plugin->getConfigPath() . 'bootstrap.php',
-            true
-        );
+        return is_file($plugin->getConfigPath() . 'bootstrap.php');
```

Comment:
This won't run the bootstrap function file anymore. What uses the return value of this?

Reply:
just tried to keeping BC without running /loading file

Reply:
also we can remove calling this method into load() and deprecating  this method in 3.6 or 3.7
I not found any usage of `Router::bootstrap()` in out of `load()`

Reply:
The point of `Plugin::bootstrap()` is to *run* the file though. If we don't run the file here, applications not using `Application` will not have their plugins load correctly.